src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/ShieldUtilitySpell.java

```diff
@@ -101,16 +116,14 @@ public void appendHoverText(ItemStack pStack, @Nullable Level pLevel, List<Compo
         pTooltipComponents.add(new TextComponent("While equipped in a ").withStyle(ChatFormatting.GRAY)
                                        .append(new TextComponent("utility slot").withStyle(ChatFormatting.DARK_PURPLE))
                                        .append(":"));
-        pTooltipComponents.add(new TextComponent("  - Stores ").withStyle(ChatFormatting.GRAY)
-                                       .append(new TextComponent(totalHits + " charges").withStyle(ChatFormatting.AQUA))
-                                       .append(", each fully negating the damage from an attack"));
-        pTooltipComponents.add(new TextComponent("  - Charges are fully replenished after ").withStyle(ChatFormatting.GRAY)
+        pTooltipComponents.add(new TextComponent("  - Adds ").withStyle(ChatFormatting.GRAY)
+                                       .append(new TextComponent(totalHits + " damage charges").withStyle(ChatFormatting.AQUA))
+                                       .append(", each absorbing the damage from an attack"));
```

Comment:
Should specify that each stack absorbs half a hearts worth of damage from an attack, and remove `damage` from `damage charges` since that doesn't clearly convey their functionality IMO.

Reply:
Fixed